### PR TITLE
prokka needs Bio-Search-hmmer, but it is not part of BioPerl>1.7.3

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/b/Bio-SearchIO-hmmer/Bio-SearchIO-hmmer-1.7.3-GCC-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+dependencies = [
+    ('Perl', '5.32.0'),
+    ('BioPerl', '1.7.8'),
+]
+
+options = {'modulename': 'Bio::SearchIO::hmmer3'}
+
+sanity_check_paths = {
+    'files': ['bin/bp_%s.pl' % x for x in ['hmmer_to_table', 'parse_hmmsearch']],
+    'dirs': ['lib/perl5/site_perl/%(perlver)s/x86_64-linux-thread-multi/auto'],
```
Comment (Micket) [inline easybuild/easyconfigs/b/Bio-SearchIO-hmmer/Bio-SearchIO-hmmer-1.7.3-GCC-10.2.0.eb:25]: Hm, are we building this? Might be best to leave out that platform specific subdirectory

**Review by Micket — DISMISSED** [review]

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 8 out of 8 (2 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/cfe491fbeaa54bb46435b909d5d86c57 for a full test report.

File: easybuild/easyconfigs/b/Bio-SearchIO-hmmer/Bio-SearchIO-hmmer-1.7.3-GCC-10.2.0.eb (hunk 1 above)
Comment (hajgato) [reply easybuild/easyconfigs/b/Bio-SearchIO-hmmer/Bio-SearchIO-hmmer-1.7.3-GCC-10.2.0.eb]: done

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 6 out of 6 (2 easyconfigs in total)
node366.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/c212169745594412a754b6f86222ffd3 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @hajgato!

**Pull request merged**